### ISSUE TITLE
Disk Size

### ISSUE THREAD

**Issue by erdog** [dduportal/boot2docker-vagrant-box#47]
Hi,

Is there a way to increase the size of the disk?

Thanks!

**Comment by dduportal** [comment]
Hello @erdog  !

It is something inside the template: so not "easily" doable from now.

The default value is the one from [Packer](https://packer.io)'s virtualbox-iso builder: https://www.packer.io/docs/builders/virtualbox-iso.html#disk_size 
It is 40 Gb by default, so you should have ~36 Gb usable.

Anyway, since this is completely tunable, it it possible to change the default value, since we have dynamic harddrives.

What would be your need in term of disk size ?

**Comment by dduportal** [comment]
Just did a quick test with Virtualbox provider:

- "disk_size" set to "120000" (~ 120 Gb)
- Box size : 34 (was 34 Mb before \o/ )
- VM size when started : 60 Mb (was: 59 Mb before)
- 115 Gb free seen within the VM : 
```bash
 vagrant ssh
                        ##         .
                  ## ## ##        ==
               ## ## ## ## ##    ===
           /"""""""""""""""""\___/ ===
      ~~~ {~~ ~~~~ ~~~ ~~~~ ~~~ ~ /  ===- ~~~
           \______ o           __/
             \    \         __/
              \____\_______/
 _                 _   ____     _            _
| |__   ___   ___ | |_|___ \ __| | ___   ___| | _____ _ __
| '_ \ / _ \ / _ \| __| __) / _` |/ _ \ / __| |/ / _ \ '__|
| |_) | (_) | (_) | |_ / __/ (_| | (_) | (__|   <  __/ |
|_.__/ \___/ \___/ \__|_____\__,_|\___/ \___|_|\_\___|_|
Boot2Docker version 1.11.1, build HEAD : 7954f54 - Wed Apr 27 16:36:45 UTC 2016
Docker version 1.11.1, build 5604cbe
docker@boot2docker:~$ df -h
Filesystem                Size      Used Available Use% Mounted on
tmpfs                     1.3G    164.5M      1.2G  12% /
tmpfs                   749.2M         0    749.2M   0% /dev/shm
/dev/sda2               115.2G     60.1M    109.3G   0% /mnt/sda2
cgroup                  749.2M         0    749.2M   0% /sys/fs/cgroup
Users_dadou_workspace_tests
                        464.8G    388.4G     76.4G  84% /Users/dadou/workspace/tests
/dev/sda2               115.2G     60.1M    109.3G   0% /mnt/sda2/var/lib/docker/aufs
```

So I'm switching to this value on master branch, it will be the default on the next release.

**Comment by dduportal** [comment]
Implemented in #49 .

You can temporary use this 1.11.1-v2 box image (stored along the other on my Dropbox, not visible in Atlas), until a new boot2docker release (1.11.2 or 1.12) comes up: https://dl.dropboxusercontent.com/u/2524496/boxes/boot2docker_virtualbox_1.11.1-v2.box 

Feel free to reopen issue if you want to discuss.
